File: app/src/test/java/com/example/moviemvp/moviedetail/MovieDetailsModelTest.java
package com.example.moviemvp.moviedetail;

import org.junit.Before;
import org.junit.Test;
import org.mockito.Mock;
import org.mockito.Mockito;
import org.mockito.MockitoAnnotations;

public class MovieDetailsModelTest {
    @Mock
    private MovieDetailsModel model;
    private MovieDetailsContract.Model.OnFinishedListener a;

    @Before
    public void setUp() {
        MockitoAnnotations.initMocks(this);
        model = new MovieDetailsModel();
    }

    @Test
    public void getMovieDetails() {
        model.getMovieDetails(a, Mockito.eq(1));
    }
}

File: app/src/test/java/com/example/moviemvp/movie_detail/MovieDetailsModelTest.kt
package com.example.moviemvp.movie_detail

import com.example.moviemvp.movie_detail.MovieDetailsContract.Model.OnFinishedListener
import org.junit.Before
import org.junit.Test

import org.junit.Assert.*
import org.mockito.Mockito
import org.mockito.MockitoAnnotations

class MovieDetailsModelTest {
    private var model:MovieDetailsModel= MovieDetailsModel()
    private lateinit var listener: OnFinishedListener
    @Before
    fun setUp() {
        MockitoAnnotations.initMocks(this)
        listener=Mockito.mock(OnFinishedListener::class.java)
    }

    @Test
    fun getMovieDetails() {
        model.getMovieDetails(listener,Mockito.eq(1))
    }
}